scripts/Makefile.am

```diff
@@ -19,6 +19,7 @@ EXTRA_DIST = snapper-hourly snapper-daily zypp-plugin.py $(pam_snapper_SCRIPTS)
 install-data-local:
 	install -D snapper-hourly $(DESTDIR)/etc/cron.hourly/suse.de-snapper
 	install -D snapper-daily $(DESTDIR)/etc/cron.daily/suse.de-snapper
+	install -D bash-completion.sh $(DESTDIR)/usr/share/bash-completion/completions/snapper
```

Comment:
You also need to add this file to EXTRA_DIST in line 17 if i am not completely wrong. Thats what I also was adviced in my boot timer PR.